🆕(build) Add home/landing featured story logic

Snugug opened the pull request:
* Codifies logic for the featured stories on the home and landing pages (available in `landingCollections.home` and `landingCollections.landing` respectively).
* Makes stories sorted by date as opposed to title and weight
* Corrects the section for a blog post that was missed with a previous PR
* Logo fix


There should be very little visual difference in the staging site for this PR

Snugug committed:
Add home and landing featured story logic

Snugug committed:
Correct blog post section

Snugug committed:
Temporary fix for logo

Snugug committed:
Move shared styling to logo scss file

This was getting spun out into page-specific CSS during build

This doesn't fully resolve the logo issue, but it cleans it up and makes the logo not appear square

Snugug committed:
True fix for logo issue

Not just inline styling, but _inline styling_

Snugug committed:
Merge branch 'redesign/2022' into feature/featured-stories

Snugug committed:
Ensure landing featured story isn't in regular story list

nickalcantara reviewed (changes requested):
LGTM, just one small call out for a function description that I think may have been deleted on accident.

From what I can tell, it seems like this allows for up to 3 featured stories on the homepage and ensures that the featured story on the landing page is _not_ one that's been featured on the homepage, which makes sense to me!

Snugug committed:
Update determineFeaturedStories description, update logic

We only want to count unique tags, not all tags (so PWA, Game, PWA is OK, whereas previous logic, it wouldn't have been)

Add fallback for home page if all featured stories have the same tags

nickalcantara commented on site/_data/eleventyComputed.js:
```diff
@@ -16,7 +16,58 @@
+  for (const c of content) {
+    const type = get(c, 'data.tags[1]');
+    if (type) {
+      if (home < 3) {
```
Should this be `home.length < 3?`

Snugug replied:
Yes. Updated.

Snugug committed:
Fix home length

Snugug committed:
Merge branch 'redesign/2022' into feature/featured-stories

Snugug committed:
Merge branch 'redesign/2022' into feature/featured-stories

Snugug committed:
Update YouTube featured image

Snugug committed:
Migrate home page case studies to new logic

Snugug committed:
Migrate stories landing page to new logic

Vanss472 reviewed (approved):
LGTM!
